simulator/config/config.go

```diff
@@ -136,23 +139,15 @@ func getPort() (int, error) {
 }
 
 // getKubeAPIServerURL gets KubeAPIServerURL from environment variable first, if empty from the config file.
-func getKubeAPIServerURL() string {
-	p := os.Getenv("KUBE_API_PORT")
-	if p == "" {
-		p = strconv.Itoa(configYaml.KubeAPIPort)
-		if p == "" {
-			p = "3131"
-		}
-	}
-
-	h := os.Getenv("KUBE_API_HOST")
-	if h == "" {
-		h = configYaml.KubeAPIHost
-		if h == "" {
-			h = "127.0.0.1"
```

Comment:
Ah, we no longer need to keep these configurations actually. But, we cannot immediately remove them because it'd be a breaking change.

For now, please add a note on the document of config that they're old configuration and will be removed very soon.